tests/test_milvus_vector_db_service.py

```diff
@@ -322,8 +321,7 @@ def test_delete_by_keys(milvus_service: MilvusVectorDBService,
 
     # Delete data by keys from the collection.
     keys_to_delete = [5, 6]
-    response = milvus_service.delete_by_keys(collection_name, keys_to_delete)
-    assert response == keys_to_delete
+    milvus_service.delete_by_keys(collection_name, keys_to_delete)
```

Comment:
Question: What changed? What gets returned from `delete_by_keys` now that it no longer returns the deleted keys?

Reply:
milvus (Embedded server version `2.3.2` for testing) recently released has a bug. It's not returning keys that are being deleted, where as with `milvus==2.3.1` (embedded  & standalone) returns list of keys as a response. So, I have removed assertion  on `delete_by_keys` response and just asserting on the query results after delete operation.

Reply:
Are we sure it's a bug and not just an API change? If so, It may be better to keep use 2.3.1 until there's a fix. If it's just an API change, then the updates make sense to me.

Reply:
I'm wondering, If it's just an API, it shouldn't even return the list. And also it has to align with the current version of `pymilvus`. Running delete operation with `pymilvus==2.3.2` and `milvus==2.3.1` returns the same result.